spec/fixtures/mock_project.md

```diff
@@ -1,45 +1,51 @@
 # Entities
 
-## Mock:Bar
+## Mock:Role
```

Comment:
@dim btw, I can fall back to `Mock::Role` double colon, just need to alter `def identifier` to produce nice single-hyphen `#mock-role`.

Even more, we might need to introduce some kind of `Apidoc.new(..., flavour: :github/:bitbucket/:etc)` as Github seems to generate header ids like `#mockrole` (from 0-9a-z only).

Reply:
I prefer that

Reply:
"that" - what? Single-colon?